src/radical/pilot/task_description.py

```diff
@@ -389,6 +394,12 @@ class TaskDescription(ru.TypedDict):
             submitted to the pilot with the given ID. If that pilot is not known
             to the TaskManager, an exception is raised.
 
+        slots (rp.Slots, optional): information on where exactly each rank of
```

Comment:
`rp.Slots` -> `radical.pilot.Slot` (thus it will be set as a link/reference)

Reply:
Thanks, done.